<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QDialog" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Уважаемый пользователь! Вы успешно подключились к базе данных музыкального магазина!</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Теперь выберете действие, которое Вы хотите произвести, работая с информационным порталом:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="showMusicButton">
     <property name="text">
      <string>Показать музыкальные исполнения заданного ансамбля или исполнителя</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="showDiscButton">
     <property name="text">
      <string>Показать пластинки заданного ансамбля или исполнителя</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QRadioButton" name="editDiscButton">
     <property name="text">
      <string>Редактировать данные о пластинках</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="addEnsembleButton">
     <property name="text">
      <string>Добавить информацию об ансамблях</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="chooseButton">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Выйти из портала</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QRadioButton" name="showRatingButton">
     <property name="text">
      <string>Показать лидеров продаж прошлого или текущего года</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
